packages/mjml-core/src/index.js

```diff
@@ -167,6 +172,9 @@ export default function mjml2html(mjml, options = {}) {
         className
       ] = `{ width:${parsedWidth}${unit} !important; }`
     },
+    addHeadSyle(componentName, headStyle) {
```

Comment:
I wonder if with an arrow function you could omit the componentName here ?

Reply:
Well I think we can keep it, might rename it as `identifier` instead  of `componentName`